docs/UserGuide.md

```diff
@@ -122,7 +122,7 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name and nric is searched.
+* Only the name or nric is searched.
```

Comment:
Technically its more accurate to say `and`, cuz both are searched at the same time, like even if you do `find S9999999F` and someones name is somehow `S9999999F`, it will still match

Anyways, maybe to be super clear, can say
```suggestion
* Only the `NAME` and `NRIC` of an elderly are searched.
```

Reply:
Is it better to remove of an elderly, because we can search NRIC of person A and name of person B at the same time.
I put 'or' because like we can search nric / name / nric and name so either one works, but like maybe 'and' makes more sense when user reads

Reply:
I think it should be and/or instead

Reply:
Ya, this is more apprioriate